src/sentry/integrations/vsts/client.py

```diff
@@ -265,3 +266,23 @@ def delete_subscription(self, instance, subscription_id):
                 subscription_id=subscription_id,
             )
         )
+
+    def search_issues(self, instance, query=None):
+        return self.post(
+            VstsApiPath.work_item_search.format(
+                account_name=instance,
+            ),
+            data={
+                'searchText': query,
+                '$top': 1000,
+                'filters': {
+                    'System.WorkItemType': [
```

Comment:
We added workitem type to get it working. I am unsure if a user can create their own types.

Reply:
this seems fine, don't think we need to support custom type if that's even possible